Michaelwwest98 UI subsystem signoff

Michaelwwest98 opened the pull request:
Hopefully this is the correct spot to create a pull request.

Michaelwwest98 committed:
Create User-Interface-Subsystem

Tried writing in Markdown code directly, seeing if it works out and creates what I want in the rendered file

Michaelwwest98 committed:
Rename User-Interface-Subsystem to User-Interface-Subsystem.md

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Michaelwwest98 commented:
Thankfully, it is the correct spot.

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

JesseTNRoberts reviewed (commented):
A purchased website is not an appropriate method. Also, this is not nearly specific enough for this design to be buildable. 


**Buildability is a requirement for systems to be considered designed and for checkpoint 2 to be met.

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Minor Analysis and BOM deletions

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Added description of computers being used to view the website

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Deleted all mentions of websites, as we decided to program software instead

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Added our laptops to the BOM

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Changed the functionality description as well as analysis to better fit our current agreed UI structure as well as minor fixes to specs and constraints and fixing the BOM considering the fact that no purchases will be necessary for this subsystem.

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Added a source for Python usage

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Michaelwwest98 committed:
Add files via upload

.dwg upload for the UI Block Diagram

Michaelwwest98 committed:
Delete Documentation/Images/UI_Block_Diagram_1.dwg

Michaelwwest98 committed:
Add files via upload

.pdf file of UI Block Diagram

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Added new block diagram, hopefully it renders properly

Michaelwwest98 committed:
Add files via upload

Added UI_Block_Diagram_1.jpg

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Added multiple more specific analytical points in the Analysis section, as well as changing the Block Diagram from a .pdf file to a .jpg file so it would show on Preview

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Deleted a stray parantheses

JesseTNRoberts reviewed (commented):
I'm unclear what this signoff is for

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Added some descriptions to both Functionality and Analysis as to why this subsystem exists.

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Added some further explanation in the functionality section as to what precisely this subsystem will do to display data to responders, as well as explaining why intuitiveness is important in the UI for the overall system

JesseTNRoberts commented on Documentation/Signoffs/User-Interface-Subsystem.md:
```diff
@@ -0,0 +1,31 @@
+# User Interface Subsystem
+
+## Functionality
```
Be brief.

JesseTNRoberts replied:
What is this subsystem going to do? Don't describe the surrounding systems.

JesseTNRoberts commented on Documentation/Signoffs/User-Interface-Subsystem.md:
```diff
@@ -0,0 +1,31 @@
+![UI_Block_Diagram_1](https://github.com/Michaelwwest98/DARPA-Drone-Triage-Sensing-System/blob/cc92902a6190693c7c66fbc01ece6b466e08f3d2/Documentation/Images/UI_Block_Diagram_1.JPG)
+
+
+## Analysis
```
I'm not reading a block of text like this. This is not analysis.

JesseTNRoberts reviewed (commented):
Are these the only constraints on the system? How fast should it update?

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Organized the Analysis section into subsections and moved some information better suited towards Analysis from the Functionality section to the Analysis section

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Added titles for each individual subsection of Analysis

Michaelwwest98 committed:
Delete Documentation/Images/UI_Block_Diagram_1.pdf

JesseTNRoberts reviewed (commented):
This isn't buildable and doesn't consider existing designs

Michaelwwest98 committed:
Delete Documentation/Images/UI_Block_Diagram_1.JPG

Michaelwwest98 committed:
Add files via upload

Replaced the Block Diagram JPG

Michaelwwest98 committed:
Delete Documentation/Images/UI_Block_Diagram_1.JPG

Michaelwwest98 committed:
Add files via upload

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Changed the Block Diagram, cut an unnecessary description of other subsystems in the Functionality section, added Constraints for Weight and Speed, and wrote Analysis sections for Weight and Speed

JesseTNRoberts reviewed (commented):
Like we talked about, I think the constraints here are too loose

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Added in a constraint and related analysis about an input to turn the system off to remain compliant with safety codes

JesseTNRoberts reviewed (commented):
Did not address issue with constraints. How fast do the various items need to update? 

Verify that all constraints from the conceptual design are here.

How is a way to turn off related to NFPA 70? And how does NFPA 70 apply?

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Added some more speed constraints, and analysis there of as well as deleted the on and off switch from the constraints section

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Added some more specific sources in regards to Conceptual Design Constraints 12 and 16

JesseTNRoberts reviewed (commented):
How does the weight constrain you?

Michaelwwest98 committed:
Update User-Interface-Subsystem.md

Removed Weight as a constraint and added some information from the table to the Analysis section instead.
